### PR TITLE
Karan m1 schrodinger

### PR DESCRIPTION
**Pull request by soltaniehha**

### CODE REVIEW

**Comment by soltaniehha** [comment]
Hi @karanprime - Great work. I have a few comments before merging your code to the main branch. 
- 

[ ] Can you change the folder name to "Learning_the_Schrödinger_Equation"?
[ ] Please move the content of the "lessons" folder into the main "Learning_the_Schrödinger_Equation" folder.

My main concern is that you have used torch as opposed to keras. What is the justification for this choice? We agreed to use tf/keras for all deep learning modules. Obviously, we don't have enough time before the workshop to fix this but we would need to address it later.

**Comment by karanprime** [comment]
Hi @soltaniehha, I have updated the folder structure to that specification. I have named it Learning_the_Schrodinger_Equation instead of Learning_the_Schrödinger_Equation because ö might cause weird bugs in folder names.

Regarding the use of PyTorch, I used it because it is much easier to define the derivatives/gradients required for the physics informed neural net in PyTorch, and the code for doing that is more readable. I can change it to Keras/TF during the workshop.
